operators/ads-operator/v22.6.0-1/manifests/ads-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,1593 @@
+          - astradsvolumesnapshots/status
+          verbs:
+          - get
+          - patch
+          - update
+        - apiGroups:
+          - ""
+          resources:
+          - events
+          verbs:
+          - create
+          - get
+          - list
+          - patch
+          - watch
+        - apiGroups:
+          - monitoring.netapp.com
+          resources:
+          - agents
+          verbs:
+          - create
+          - delete
+          - get
+          - list
+          - patch
+          - watch
+        serviceAccountName: default
+      deployments:
+      - name: astrads-operator
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              control-plane: operator
+              product: astrads
+          strategy:
+            type: Recreate
+          template:
+            metadata:
+              labels:
+                control-plane: operator
+                product: astrads
+            spec:
+              containers:
+              - command:
+                - /operator
+                image: registry.connect.redhat.com/netapp/ads-operator@sha256:ed0b61939726f8c88952ed09ac64a60948fce8563feac814c805543b9e24f7e8
+                imagePullPolicy: IfNotPresent
+                livenessProbe:
+                  exec:
+                    command:
+                    - /bin/sh
+                    - -c
+                    - liveness -heartbeat 30
+                  failureThreshold: 3
+                  initialDelaySeconds: 30
+                  periodSeconds: 15
+                  successThreshold: 1
+                  timeoutSeconds: 15
+                name: manager
+                ports:
+                - containerPort: 443
+                  name: webhook-server
+                  protocol: TCP
+                readinessProbe:
+                  exec:
+                    command:
+                    - cat
+                    - /tmp/k8s-webhook-server/serving-certs/tls.crt
+                  failureThreshold: 5
+                  initialDelaySeconds: 20
+                  successThreshold: 1
+                  timeoutSeconds: 5
+                resources:
+                  limits:
+                    cpu: 200m
+                    memory: 150Mi
+                  requests:
+                    cpu: 100m
+                    memory: 50Mi
+                securityContext:
+                  capabilities:
+                    drop:
+                    - ALL
+                  readOnlyRootFilesystem: true
+                volumeMounts:
+                - mountPath: /data/probe
+                  name: ephemeral
+                - mountPath: /templog
+                  name: log-dir
+                - mountPath: /tmp/k8s-webhook-server/serving-certs
+                  name: webhook-cert
+              priorityClassName: system-cluster-critical
+              securityContext:
+                runAsNonRoot: true
+                runAsUser: 20000
```

Comment:
OpenShift assigns a range of UID's per namespace for the entities in that namespace to use. If you need to keep the `runAsUser: 20000` you will need to provide the proper SCC/RBAC/etc to make everything run as you have defined.